index.html

```diff
@@ -17,26 +17,37 @@
         <nav class="burger-icon">
             <i class="fa-solid fa-bars"></i>
         </nav>
-    </header>
-    <section class="hero">
-        <div class="container hero-text">
-            <h1>I'm David, Glad to see you</h1>
-            <p>I'm a software developer, I can help you build a product, 
-            feature or Website. Look trough some of my work and experience, 
-            if you like what you see and have a project you need coded, don't
-            hesitate to contact me. 
-            </p>
-            <span>LET'S CONNECT</span>
-            <ul class="social-media">
-                <li><a href="https://twitter.com/DavidEspinoG"><img src="./img/twitter.png" alt="twitter icon"></a></li>
-                <li><a href="https://www.linkedin.com/in/davidespinog/"><img src="./img/linkedin.png" alt="linkedin icon"></a></li>
-                <li><a href="#"><img src="./img/M.png" alt="M icon"></a></li>
-                <li><a href="https://github.com/DavidEspinoG"><img src="./img/git.png" alt="github icon"></a></li>
-                <li><a href="#"><img src="./img/hand.png" alt="hand icon"></a></li>
+        <nav class="desktop-navigation">
+            <ul>
+                <li><a class="nav-link" href="#portfolio">Portfolio</a></li>
+                <li><a class="nav-link" href="#about">About</a></li>
+                <li><a class="nav-link" href="#contact">Contact</a></li>
             </ul>
-        </div>
-    </section>
-    <section class="cards-container">
+        </nav>
+    </header>
+    <div class="hero-background">
+        <section class="hero">
+            <div class="container hero-text">
+                <h1>I'm David, Glad to see you</h1>
```

Comment:
- [ ] Kindly follow the design pattern of the Figma template for the `h1` tag. If you look at the template, the text _**'Glad to See You!**_ is beneath the name text. Kindly implement this change because your design should be identical to that of the template. 💻 

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/213260615-ad0c2e2d-b669-43ba-b3f6-21605ff31fad.png)

### Your Design

![image](https://user-images.githubusercontent.com/77106962/213260684-c83e5e12-a2fd-48dc-b07c-b9685d4c3ab3.png)